recipes/nanobench/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "nanobench-static" %}
```

```suggestion
{% set name = "nanobench" %}
```

Please switch to the header-only distribution of nanobench. Instead of the static library version because this library is advertised as "single-header" and the static version of the library includes a third-party library which you haven't included the license for.

In general, our channel strongly discourages providing static library objects because they are less efficient for our channel as a whole.